colcon-build-and-test/action.yaml

```diff
@@ -72,7 +72,7 @@ runs:
         colcon test --event-handlers console_cohesion+ \
           --packages-above ${{ inputs.target-packages }} \
           --return-code-on-test-failure
-        colcon lcov-result --packages-above ${{ inputs.target-packages }}
+        (find build -name coverage.info | grep .) && colcon lcov-result --packages-above ${{ inputs.target-packages }}
```

Comment:
I feel this is easier to understand for me, how about this?
- Use if
- Add `/` to `build`
- Use `test -f`

```bash
if [ -f build/coverage.info ]; then 
  colcon lcov-result --packages-above ${{ inputs.target-packages }}
fi
```

Reply:
What does Add / to build and Use test -f mean? Please explain it :pray:

Reply:
It can explicitly explain:
- It's not a subcommand.
- It's a directory.

Reply:
`test -f` is `[ -f ]`

Reply:
Ah, I understood. Thanks!